examples/stargate-prometheus-grafana/start_stargate-prometheus-grafana.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+COMPOSE_FILE=stargate-prometheus-grafana.yml
```

Comment:
Just FYI, naming the file "docker-compose.yaml" means you don't have to provide it in commands

Reply:
I changed the file to the default "docker-compose.yml".